08_deploy_bmo.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/bash
+
+source bmo_install_prereqs.sh
+# Get the latest bits for baremetal-operator
+go get github.com/metalkube/baremetal-operator
+export BMOPATH=~/go/src/github.com/metalkube/baremetal-operator
```

Comment:
This should use GOPATH not ~

Reply:
And similar to the comment below we could use 03_ocp_repo_sync.sh to get the repo vs go get, the original idea was that we'd clone everything needed there so that test iterations on the other scripts during development could be faster.